x-pack/plugins/cloud_security_posture/server/routes/configuration/update_rules_configuration.test.ts

```diff
@@ -175,37 +176,106 @@ describe('Update rules configuration API', () => {
 
   it('validate adding new data.yaml to package policy instance', async () => {
     const packagePolicy = createPackagePolicyMock();
+    packagePolicy.vars = { dataYaml: { type: 'yaml' } };
 
     const dataYaml = 'data_yaml:\n  activated_rules:\n  cis_k8s:\n    - 1.1.1\n    - 1.1.2\n';
     const updatedPackagePolicy = setVarToPackagePolicy(packagePolicy, dataYaml);
-
-    expect(updatedPackagePolicy.vars).toEqual({ dataYaml: { type: 'config', value: dataYaml } });
+    expect(updatedPackagePolicy.vars).toEqual({ dataYaml: { type: 'yaml', value: dataYaml } });
   });
 
-  it('validate updatePackagePolicy is called with the right parameters', async () => {
+  it('validate updateAgentConfiguration is called with the right parameters', async () => {
```

Comment:
the test description is wrong. 
On line 217 it explicitly call `updateAgentConfiguration`. 
So what exactly we are testing here?
Did you meant that the tests checks `updateAgentConfiguration` in a specific condition?